docs/configuration-files/add-configuration-file.md

```diff
@@ -21,3 +21,4 @@ links:
 1. Use the drop-down menu to choose `CONFIGURATION FILE`
 1. On the right corner of the list click `ADD CONFIGURATION FILE`
 1. Follow the instructions in the add configuration file step form then click `CONFIRM`
+    > A new deployment is required for your changes to take effect, follow the guide [Deploy Application](/docs/applications/deploy-application/)
```

Comment:
I believe we're using commas instead of dots and I also see that in other documentation topics. IMHO reader should be given time to breath and digest info in short sentences, so they can be easier to understand.

Possible approaches:
1) Just replacing commas with dots, allowing readers to breathe
```
> A new deployment is required for your changes to take effect. Follow the guide [Deploy Application](/docs/applications/deploy-application/)
```

2) Make an assertion (a deployment is required) then, as a separate note, offer a way to find out about how to deploy the application, for readers still not familiar with that concept:
```
> A new deployment is required for your changes to take effect.
  - For more information, see [Deploy Application](/docs/applications/deploy-application/)
```